native image generation fails with 3.3.0 works with 3.2.1

### Issue description

Native image generation for the same 3.3.4 application fails with [Micronaut Gradle Plugin 3.3.0](https://plugins.gradle.org/plugin/io.micronaut.minimal.application) and works with 3.2.1

```
./gradlew nativeCompile --scan

> Task :compileJava
Note: Creating bean classes for 3 type elements

> Task :generateResourcesConfigFile
[native-image-plugin] Resources configuration written into /Users/sdelamo/Downloads/demo 78/build/native/generated/generateResourcesConfigFile/resource-config.json

> Task :nativeCompile FAILED
[native-image-plugin] Toolchain detection is disabled, will use GraalVM from /Users/sdelamo/.sdkman/candidates/java/22.0.0.2.r17-grl.
[native-image-plugin] Using executable path: /Users/sdelamo/.sdkman/candidates/java/22.0.0.2.r17-grl/bin/native-image
Error: Found unrecognized options while parsing argument file '/var/folders/pb/dx1ms2_92_g54v9jvzv7k4hc0000gn/T/native-image16542058272032056823args':
78/build/libs/demo-0.1.jar:/Users/sdelamo/.gradle/caches/modules-2/files-2.1/io.micronaut/micronaut-http-client/3.3.4/808288af06565cbf1a5faadd754109d4e4ecf5e2/micronaut-http-client-3.3.4.jar:/Users/sdelamo/.gradle/caches/modules-2/files-2.1/io.micronaut/micronaut-http-server-netty/3.3.4/5a8f9b7138d1e424f773eb9529ead6b9ddc73213/micronaut-http-server-netty-3.3.4.jar:/Users/sdelamo/.gradle/caches/modules-2/files-2.1/io.micronaut/micronaut-http-netty/3.3.4/410880b340a62db37c885841469859eb78c3636c/micronaut-http-netty-3.3.4.jar:/Users/sdelamo/.gradle/caches/modules-2/files-2.1/io.micronaut/micronaut-http-server/3.3.4/98653658215ec21a1656061ecf94dc53f15ed43b/micronaut-http-server-3.3.4.jar:/Users/sdelamo/.gradle/caches/modules-2/files-2.1/io.micronaut/micronaut-websocket/3.3.4/14cb66367eda76cac323a3ae6c1cab966afb5a8b/micronaut-websocket-3.3.4.jar:/Users/sdelamo/.gradle/caches/modules-2/files-2.1/io.micronaut/micronaut-http-client-core/3.3.4/b213ef4c6a5a68a55dcbc1b911431c975d570605/micronaut-http-client-core-3.3.4.jar:/Users/sdelamo/.gradle/caches/modules-2/files-2.1/io.micronaut/micronaut-runtime/3.3.4/cafb8035b868c94226abb9374909286e55a83a10/micronaut-runtime-3.3.4.jar:/Users/sdelamo/.gradle/caches/modules-2/files-2.1/io.micronaut/micronaut-jackson-databind/3.3.4/880a8dd1b12903958e2105e0fac4b750f0af1d73/micronaut-jackson-databind-3.3.4.jar:/Users/sdelamo/.gradle/caches/modules-2/files-2.1/io.micronaut/micronaut-validation/3.3.4/99520e29a847d19a23af4a7e81e4719a0bd9d33f/micronaut-validation-3.3.4.jar:/Users/sdelamo/.gradle/caches/modules-2/files-2.1/io.micronaut/micronaut-router/3.3.4/2e1625b732eb872eb0cd6da8461ed7470ec34b2d/micronaut-router-3.3.4.jar:/Users/sdelamo/.gradle/caches/modules-2/files-2.1/io.micronaut/micronaut-jackson-core/3.3.4/d838c891c766adadde3e73daf0a0aa1ff0a26ccb/micronaut-jackson-core-3.3.4.jar:/Users/sdelamo/.gradle/caches/modules-2/files-2.1/io.micronaut/micronaut-json-core/3.3.4/29c74df7b925d7479e03a1010a3ddc428f2c0de1/micronaut-json-core-3.3.4.jar:/Users/sdelamo/.gradle/caches/modules-2/files-2.1/io.micronaut/micronaut-http/3.3.4/326792ef810fb698343ba25ad0b4041261165f2e/micronaut-http-3.3.4.jar:/Users/sdelamo/.gradle/caches/modules-2/files-2.1/io.micronaut/micronaut-context/3.3.4/a7e7e5208ba698f83a51961761e13df35b9e0205/micronaut-context-3.3.4.jar:/Users/sdelamo/.gradle/caches/modules-2/files-2.1/io.micronaut/micronaut-aop/3.3.4/8bed82283762f4537183bb8146bbecefe95b7bfe/micronaut-aop-3.3.4.jar:/Users/sdelamo/.gradle/caches/modules-2/files-2.1/io.micronaut/micronaut-buffer-netty/3.3.4/438ee7ebdd44c334885b00a65e524442b0370e3/micronaut-buffer-netty-3.3.4.jar:/Users/sdelamo/.gradle/caches/modules-2/files-2.1/io.micronaut/micronaut-inject/3.3.4/930b3351318428453d2677d628f66f0931c10371/micronaut-inject-3.3.4.jar:/Users/sdelamo/.gradle/caches/modules-2/files-2.1/jakarta.annotation/jakarta.annotation-api/2.0.0/f3cd84cc45f583a0fdc42a8156d6c5b98d625c1a/jakarta.annotation-api-2.0.0.jar:/Users/sdelamo/.gradle/caches/modules-2/files-2.1/ch.qos.logback/logback-classic/1.2.10/f69d97ef3335c6ab82fc21dfb77ac613f90c1221/logback-classic-1.2.10.jar:/Users/sdelamo/.gradle/caches/modules-2/files-2.1/com.fasterxml.jackson.datatype/jackson-datatype-jdk8/2.12.6/ccbbe6e72e3665e7ad9d0eee4613e1d8a09b81ae/jackson-datatype-jdk8-2.12.6.jar:/Users/sdelamo/.gradle/caches/modules-2/files-2.1/com.fasterxml.jackson.datatype/jackson-datatype-jsr310/2.12.6/f7d0d854f24c4254885c275a09fb885ef578b48/jackson-datatype-jsr310-2.12.6.jar:/Users/sdelamo/.gradle/caches/modules-2/files-2.1/com.fasterxml.jackson.core/jackson-databind/2.12.6/fac216b606c1086e36acea6e572ee61572ad1670/jackson-databind-2.12.6.jar:/Users/sdelamo/.gradle/caches/modules-2/files-2.1/com.fasterxml.jackson.core/jackson-annotations/2.12.6/9487231edd6b0b1f14692c9cba9e0462809215d1/jackson-annotations-2.12.6.jar:/Users/sdelamo/.gradle/caches/modules-2/files-2.1/com.fasterxml.jackson.core/jackson-core/2.12.6/5bf206c0b5982cfcd868b3d9349dc5190db8bab5/jackson-core-2.12.6.jar:/Users/sdelamo/.gradle/caches/modules-2/files-2.1/javax.annotation/javax.annotation-api/1.3.2/934c04d3cfef185a8008e7bf34331b79730a9d43/javax.annotation-api-1.3.2.jar:/Users/sdelamo/.gradle/caches/modules-2/files-2.1/io.netty/netty-handler-proxy/4.1.73.Final/d1afa6876c3d3bdbdbe5127ddd495e6514d6e600/netty-handler-proxy-4.1.73.Final.jar:/Users/sdelamo/.gradle/caches/modules-2/files-2.1/io.netty/netty-codec-http2/4.1.73.Final/eb145bc31fd32a20fd2a3e8b30736d2e0248b0c/netty-codec-http2-4.1.73.Final.jar:/Users/sdelamo/.gradle/caches/modules-2/files-2.1/io.netty/netty-codec-http/4.1.73.Final/1ceeac4429b9bd517dc05e376a144bbe6b6bd038/netty-codec-http-4.1.73.Final.jar:/Users/sdelamo/.gradle/caches/modules-2/files-2.1/io.netty/netty-codec-socks/4.1.73.Final/cefa44d8f5dcaab21179d945f12b6c6d7325cce9/netty-codec-socks-4.1.73.Final.jar:/Users/sdelamo/.gradle/caches/modules-2/files-2.1/io.netty/netty-handler/4.1.73.Final/1a2231c0074f88254865c3769a4b5842939ea04d/netty-handler-4.1.73.Final.jar:/Users/sdelamo/.gradle/caches/modules-2/files-2.1/io.netty/netty-codec/4.1.73.Final/9496a30a349863a4c6fa10d5c36b4f3b495d3a31/netty-codec-4.1.73.Final.jar:/Users/sdelamo/.gradle/caches/modules-2/files-2.1/io.netty/netty-transport/4.1.73.Final/abb155ddff196ccedfe85b810d4b9375ef85fcfa/netty-transport-4.1.73.Final.jar:/Users/sdelamo/.gradle/caches/modules-2/files-2.1/io.netty/netty-buffer/4.1.73.Final/244a569c9aae973f6f485ac9801d79c1eca36daa/netty-buffer-4.1.73.Final.jar:/Users/sdelamo/.gradle/caches/modules-2/files-2.1/io.projectreactor/reactor-core/3.4.14/78549cb21d2ca677807ac2863600bdaeb661601a/reactor-core-3.4.14.jar:/Users/sdelamo/.gradle/caches/modules-2/files-2.1/io.micronaut/micronaut-core-reactive/3.3.4/23c0228adeeae5dd2a01102f61d2e3e5ece5d329/micronaut-core-reactive-3.3.4.jar:/Users/sdelamo/.gradle/caches/modules-2/files-2.1/org.reactivestreams/reactive-streams/1.0.3/d9fb7a7926ffa635b3dcaa5049fb2bfa25b3e7d0/reactive-streams-1.0.3.jar:/Users/sdelamo/.gradle/caches/modules-2/files-2.1/io.micronaut/micronaut-core/3.3.4/413e8b8cbee4e42fddcb258f6aa585fd1a887bc5/micronaut-core-3.3.4.jar:/Users/sdelamo/.gradle/caches/modules-2/files-2.1/org.slf4j/slf4j-api/1.7.32/cdcff33940d9f2de763bc41ea05a0be5941176c3/slf4j-api-1.7.32.jar:/Users/sdelamo/.gradle/caches/modules-2/files-2.1/org.yaml/snakeyaml/1.30/8fde7fe2586328ac3c68db92045e1c8759125000/snakeyaml-1.30.jar:/Users/sdelamo/.gradle/caches/modules-2/files-2.1/javax.validation/validation-api/2.0.1.Final/cb855558e6271b1b32e716d24cb85c7f583ce09e/validation-api-2.0.1.Final.jar:/Users/sdelamo/.gradle/caches/modules-2/files-2.1/io.netty/netty-resolver/4.1.73.Final/bfe83710f0c1739019613e81a06101020ca65def/netty-resolver-4.1.73.Final.jar:/Users/sdelamo/.gradle/caches/modules-2/files-2.1/io.netty/netty-common/4.1.73.Final/27731b58d741b6faa6a00fa3285e7a55cc47be01/netty-common-4.1.73.Final.jar:/Users/sdelamo/.gradle/caches/modules-2/files-2.1/io.netty/netty-tcnative-classes/2.0.46.Final/9937a832d9c19861822d345b48ced388b645aa5f/netty-tcnative-classes-2.0.46.Final.jar:/Users/sdelamo/.gradle/caches/modules-2/files-2.1/ch.qos.logback/logback-core/1.2.10/5328406bfcae7bcdcc86810fcb2920d2c297170d/logback-core-1.2.10.jar:/Users/sdelamo/.gradle/caches/modules-2/files-2.1/jakarta.inject/jakarta.inject-api/2.0.1/4c28afe1991a941d7702fe1362c365f0a8641d1e/jakarta.inject-api-2.0.1.jar
78/build/native/nativeCompile
78/build/native/generated/generateResourcesConfigFile

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':nativeCompile'.
> Process 'command '/Users/sdelamo/.sdkman/candidates/java/22.0.0.2.r17-grl/bin/native-image'' finished with non-zero exit value 1

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 1s
5 actionable tasks: 3 executed, 2 up-to-date

Publishing a build scan to scans.gradle.com requires accepting the Gradle Terms of Service defined at https://gradle.com/terms-of-service. Do you accept these terms? [yes, no] yes

Gradle Terms of Service accepted.
https://gradle.com/s/awwmtqdfx2zdo
```

It fails to authenticate: 

```
% ./gradlew nativeCompile

FAILURE: Build failed with an exception.

* Where:
Settings file '/Users/sdelamo/Downloads/demo 83/settings.gradle' line: 9

* What went wrong:
A problem occurred evaluating settings 'demo'.
> Unable to clone repository contents: git@github.com:graalvm/native-build-tools.git: Auth fail

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.

* Get more help at https://help.gradle.org
```

```
 % ./gradlew nativeCompile       

> Task :settings-plugins:compileKotlin
'compileJava' task (current target is 17) and 'compileKotlin' task (current target is 1.8) jvm target compatibility should be set to the same Java version.

FAILURE: Build failed with an exception.

* Where:
Settings file '/Users/sdelamo/Downloads/demo 83/checkouts/native-build-tools/build-logic/aggregator/settings.gradle.kts' line: 46

* What went wrong:
An exception occurred applying plugin request [id: 'org.graalvm.build.common']
> Failed to apply plugin 'org.graalvm.build.common'.
   > Using dependency catalogs requires the activation of the matching feature preview.
     See the documentation at https://docs.gradle.org/7.3.1/userguide/platforms.html#sub:central-declaration-of-dependencies

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.

* Get more help at https://help.gradle.org
```

I still could  not generate a native image if I use Gradle  7.4.1 and the following `settings.gradle` 

```
plugins {
    id 'me.champeau.includegit' version '0.1.5'
}


rootProject.name="demo"

gitRepositories {
    include('native-build-tools') {
        uri = "https://github.com/graalvm/native-build-tools.git"
        branch = 'cc/rework-argfile'
    }
}
```


error:
```
> Task :nativeCompile FAILED
[native-image-plugin] Toolchain detection is disabled, will use GraalVM from /Users/sdelamo/.sdkman/candidates/java/22.0.0.2.r17-grl.
[native-image-plugin] Using executable path: /Users/sdelamo/.sdkman/candidates/java/22.0.0.2.r17-grl/bin/native-image
Error: Found unrecognized options while parsing argument file '/var/folders/pb/dx1ms2_92_g54v9jvzv7k4hc0000gn/T/native-image11088203496092318371args':
83/build/libs/demo-0.1.jar:/Users/sdelamo/.gradle/caches/modules-2/files-2.1/io.micronaut/micronaut-http-client/3.3.4/808288af06565cbf1a5faadd754109d4e4ecf5e2/micronaut-http-client-3.3.4.jar:/Users/sdelamo/.gradle/caches/modules-2/files-2.1/io.micronaut/micronaut-http-server-netty/3.3.4/5a8f9b7138d1e424f773eb9529ead6b9ddc73213/micronaut-http-server-netty-3.3.4.jar:/Users/sdelamo/.gradle/caches/modules-2/files-2.1/io.micronaut/micronaut-http-netty/3.3.4/410880b340a62db37c885841469859eb78c3636c/micronaut-http-netty-3.3.4.jar:/Users/sdelamo/.gradle/caches/modules-2/files-2.1/io.micronaut/micronaut-http-server/3.3.4/98653658215ec21a1656061ecf94dc53f15ed43b/micronaut-http-server-3.3.4.jar:/Users/sdelamo/.gradle/caches/modules-2/files-2.1/io.micronaut/micronaut-websocket/3.3.4/14cb66367eda76cac323a3ae6c1cab966afb5a8b/micronaut-websocket-3.3.4.jar:/Users/sdelamo/.gradle/caches/modules-2/files-2.1/io.micronaut/micronaut-http-client-core/3.3.4/b213ef4c6a5a68a55dcbc1b911431c975d570605/micronaut-http-client-core-3.3.4.jar:/Users/sdelamo/.gradle/caches/modules-2/files-2.1/io.micronaut/micronaut-runtime/3.3.4/cafb8035b868c94226abb9374909286e55a83a10/micronaut-runtime-3.3.4.jar:/Users/sdelamo/.gradle/caches/modules-2/files-2.1/io.micronaut/micronaut-jackson-databind/3.3.4/880a8dd1b12903958e2105e0fac4b750f0af1d73/micronaut-jackson-databind-3.3.4.jar:/Users/sdelamo/.gradle/caches/modules-2/files-2.1/io.micronaut/micronaut-validation/3.3.4/99520e29a847d19a23af4a7e81e4719a0bd9d33f/micronaut-validation-3.3.4.jar:/Users/sdelamo/.gradle/caches/modules-2/files-2.1/io.micronaut/micronaut-router/3.3.4/2e1625b732eb872eb0cd6da8461ed7470ec34b2d/micronaut-router-3.3.4.jar:/Users/sdelamo/.gradle/caches/modules-2/files-2.1/io.micronaut/micronaut-jackson-core/3.3.4/d838c891c766adadde3e73daf0a0aa1ff0a26ccb/micronaut-jackson-core-3.3.4.jar:/Users/sdelamo/.gradle/caches/modules-2/files-2.1/io.micronaut/micronaut-json-core/3.3.4/29c74df7b925d7479e03a1010a3ddc428f2c0de1/micronaut-json-core-3.3.4.jar:/Users/sdelamo/.gradle/caches/modules-2/files-2.1/io.micronaut/micronaut-http/3.3.4/326792ef810fb698343ba25ad0b4041261165f2e/micronaut-http-3.3.4.jar:/Users/sdelamo/.gradle/caches/modules-2/files-2.1/io.micronaut/micronaut-context/3.3.4/a7e7e5208ba698f83a51961761e13df35b9e0205/micronaut-context-3.3.4.jar:/Users/sdelamo/.gradle/caches/modules-2/files-2.1/io.micronaut/micronaut-aop/3.3.4/8bed82283762f4537183bb8146bbecefe95b7bfe/micronaut-aop-3.3.4.jar:/Users/sdelamo/.gradle/caches/modules-2/files-2.1/io.micronaut/micronaut-buffer-netty/3.3.4/438ee7ebdd44c334885b00a65e524442b0370e3/micronaut-buffer-netty-3.3.4.jar:/Users/sdelamo/.gradle/caches/modules-2/files-2.1/io.micronaut/micronaut-inject/3.3.4/930b3351318428453d2677d628f66f0931c10371/micronaut-inject-3.3.4.jar:/Users/sdelamo/.gradle/caches/modules-2/files-2.1/jakarta.annotation/jakarta.annotation-api/2.0.0/f3cd84cc45f583a0fdc42a8156d6c5b98d625c1a/jakarta.annotation-api-2.0.0.jar:/Users/sdelamo/.gradle/caches/modules-2/files-2.1/ch.qos.logback/logback-classic/1.2.10/f69d97ef3335c6ab82fc21dfb77ac613f90c1221/logback-classic-1.2.10.jar:/Users/sdelamo/.gradle/caches/modules-2/files-2.1/com.fasterxml.jackson.datatype/jackson-datatype-jdk8/2.12.6/ccbbe6e72e3665e7ad9d0eee4613e1d8a09b81ae/jackson-datatype-jdk8-2.12.6.jar:/Users/sdelamo/.gradle/caches/modules-2/files-2.1/com.fasterxml.jackson.datatype/jackson-datatype-jsr310/2.12.6/f7d0d854f24c4254885c275a09fb885ef578b48/jackson-datatype-jsr310-2.12.6.jar:/Users/sdelamo/.gradle/caches/modules-2/files-2.1/com.fasterxml.jackson.core/jackson-databind/2.12.6/fac216b606c1086e36acea6e572ee61572ad1670/jackson-databind-2.12.6.jar:/Users/sdelamo/.gradle/caches/modules-2/files-2.1/com.fasterxml.jackson.core/jackson-annotations/2.12.6/9487231edd6b0b1f14692c9cba9e0462809215d1/jackson-annotations-2.12.6.jar:/Users/sdelamo/.gradle/caches/modules-2/files-2.1/com.fasterxml.jackson.core/jackson-core/2.12.6/5bf206c0b5982cfcd868b3d9349dc5190db8bab5/jackson-core-2.12.6.jar:/Users/sdelamo/.gradle/caches/modules-2/files-2.1/javax.annotation/javax.annotation-api/1.3.2/934c04d3cfef185a8008e7bf34331b79730a9d43/javax.annotation-api-1.3.2.jar:/Users/sdelamo/.gradle/caches/modules-2/files-2.1/io.netty/netty-handler-proxy/4.1.73.Final/d1afa6876c3d3bdbdbe5127ddd495e6514d6e600/netty-handler-proxy-4.1.73.Final.jar:/Users/sdelamo/.gradle/caches/modules-2/files-2.1/io.netty/netty-codec-http2/4.1.73.Final/eb145bc31fd32a20fd2a3e8b30736d2e0248b0c/netty-codec-http2-4.1.73.Final.jar:/Users/sdelamo/.gradle/caches/modules-2/files-2.1/io.netty/netty-codec-http/4.1.73.Final/1ceeac4429b9bd517dc05e376a144bbe6b6bd038/netty-codec-http-4.1.73.Final.jar:/Users/sdelamo/.gradle/caches/modules-2/files-2.1/io.netty/netty-codec-socks/4.1.73.Final/cefa44d8f5dcaab21179d945f12b6c6d7325cce9/netty-codec-socks-4.1.73.Final.jar:/Users/sdelamo/.gradle/caches/modules-2/files-2.1/io.netty/netty-handler/4.1.73.Final/1a2231c0074f88254865c3769a4b5842939ea04d/netty-handler-4.1.73.Final.jar:/Users/sdelamo/.gradle/caches/modules-2/files-2.1/io.netty/netty-codec/4.1.73.Final/9496a30a349863a4c6fa10d5c36b4f3b495d3a31/netty-codec-4.1.73.Final.jar:/Users/sdelamo/.gradle/caches/modules-2/files-2.1/io.netty/netty-transport/4.1.73.Final/abb155ddff196ccedfe85b810d4b9375ef85fcfa/netty-transport-4.1.73.Final.jar:/Users/sdelamo/.gradle/caches/modules-2/files-2.1/io.netty/netty-buffer/4.1.73.Final/244a569c9aae973f6f485ac9801d79c1eca36daa/netty-buffer-4.1.73.Final.jar:/Users/sdelamo/.gradle/caches/modules-2/files-2.1/io.projectreactor/reactor-core/3.4.14/78549cb21d2ca677807ac2863600bdaeb661601a/reactor-core-3.4.14.jar:/Users/sdelamo/.gradle/caches/modules-2/files-2.1/io.micronaut/micronaut-core-reactive/3.3.4/23c0228adeeae5dd2a01102f61d2e3e5ece5d329/micronaut-core-reactive-3.3.4.jar:/Users/sdelamo/.gradle/caches/modules-2/files-2.1/org.reactivestreams/reactive-streams/1.0.3/d9fb7a7926ffa635b3dcaa5049fb2bfa25b3e7d0/reactive-streams-1.0.3.jar:/Users/sdelamo/.gradle/caches/modules-2/files-2.1/io.micronaut/micronaut-core/3.3.4/413e8b8cbee4e42fddcb258f6aa585fd1a887bc5/micronaut-core-3.3.4.jar:/Users/sdelamo/.gradle/caches/modules-2/files-2.1/org.slf4j/slf4j-api/1.7.32/cdcff33940d9f2de763bc41ea05a0be5941176c3/slf4j-api-1.7.32.jar:/Users/sdelamo/.gradle/caches/modules-2/files-2.1/org.yaml/snakeyaml/1.30/8fde7fe2586328ac3c68db92045e1c8759125000/snakeyaml-1.30.jar:/Users/sdelamo/.gradle/caches/modules-2/files-2.1/javax.validation/validation-api/2.0.1.Final/cb855558e6271b1b32e716d24cb85c7f583ce09e/validation-api-2.0.1.Final.jar:/Users/sdelamo/.gradle/caches/modules-2/files-2.1/io.netty/netty-resolver/4.1.73.Final/bfe83710f0c1739019613e81a06101020ca65def/netty-resolver-4.1.73.Final.jar:/Users/sdelamo/.gradle/caches/modules-2/files-2.1/io.netty/netty-common/4.1.73.Final/27731b58d741b6faa6a00fa3285e7a55cc47be01/netty-common-4.1.73.Final.jar:/Users/sdelamo/.gradle/caches/modules-2/files-2.1/io.netty/netty-tcnative-classes/2.0.46.Final/9937a832d9c19861822d345b48ced388b645aa5f/netty-tcnative-classes-2.0.46.Final.jar:/Users/sdelamo/.gradle/caches/modules-2/files-2.1/ch.qos.logback/logback-core/1.2.10/5328406bfcae7bcdcc86810fcb2920d2c297170d/logback-core-1.2.10.jar:/Users/sdelamo/.gradle/caches/modules-2/files-2.1/jakarta.inject/jakarta.inject-api/2.0.1/4c28afe1991a941d7702fe1362c365f0a8641d1e/jakarta.inject-api-2.0.1.jar
83/build/native/nativeCompile
83/build/native/generated/generateResourcesConfigFile

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':nativeCompile'.
> Process 'command '/Users/sdelamo/.sdkman/candidates/java/22.0.0.2.r17-grl/bin/native-image'' finished with non-zero exit value 1

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.

* Get more help at https://help.gradle.org


```




Aside: [I upgraded starter to 7.4.1 yesterday](https://github.com/micronaut-projects/micronaut-starter/pull/1094).

--- a/gradle/libs.versions.toml
+++ b/gradle/libs.versions.toml
@@ -5,7 +5,7 @@ diffplug = "3.33.2"
 shadow = "7.1.2"
 groovy = "3.0.9"
 spock = "2.0-groovy-3.0"
-graalvmPlugin = "0.9.10"
+graalvmPlugin = "0.9.11"
 micronaut = "3.2.0"
 micronaut-aot = "1.0.0"
 log4j2 = { require = "2.17.1", reject = ["]0, 2.17["] }

--- a/src/docs/asciidoc/index.adoc
+++ b/src/docs/asciidoc/index.adoc
@@ -1,5 +1,5 @@
 = Micronaut Gradle plugin
-:native-build-tools-version: 0.9.10
+:native-build-tools-version: 0.9.11
 :kotlin-version: 1.5.30
 :micronaut-version: 3.1.1
 :gradle-version: 7.2

